wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -233,12 +233,13 @@ public static Command race(Command... commands) {
    * the others.
    *
    * @param deadline the deadline command
-   * @param commands the commands to include
+   * @param otherCommands the other commands to include
```

Comment:
I'm not seeing much benefit in this rename... But I'm not necessarily against it.

Reply:
I was concerned people might think that the `commands` argument should include the deadline, so I wanted to make it clear.